### ISSUE TITLE
Dataset release plan

### ISSUE THREAD

**Issue by 1874Mercury** [tiwarylab/GrASP#4]
Nice work! However, I noticed that there is currently no publicly available dataset for download. Would it be possible to make the dataset open and accessible?

**Comment by zwsmith200** [comment]
Yes, we will release the data at varying levels of pre-processing so you can use it to reproduce the results, play around with GNN architectures, or evaluate a completely different model using the structures split into receptor/ligand(s).

**Comment by 1874Mercury** [comment]
Great!  When can we expect the dataset to be released?

**Comment by zw-SIMM** [comment]
> Yes, we will release the data at varying levels of pre-processing so you can use it to reproduce the results, play around with GNN architectures, or evaluate a completely different model using the structures split into receptor/ligand(s).

Great work! Please release the data as soon as possible!

**Comment by Ehrenfestz** [comment]
Enlightening work!Expecting the  publishment of the dataset!
